vk_requests/session.py

```diff
@@ -46,6 +48,9 @@ def __init__(self, app_id=None, user_login=None, user_password=None,
         self.interactive = interactive
         self._access_token = None
         self._api_version = api_version
+        self._client_secret = client_secret
+        self._two_fa_supported = two_fa_supported
+        self.two_fa_force_sms = two_fa_force_sms
```

Comment:
Sorry for not suggesting it in the same comment, but to keep it consistent let's make `two_fa_force_sms` also "private"
```suggestion
        self._two_fa_force_sms = two_fa_force_sms
```